### PR TITLE
Complete support for French, Spanish and German.

### PR DESCRIPTION
**Pull request by charlesgiroux**
The switch statement for locale was missing options for French, Spanish and German.

- Added French, Spanish and German to switch statement.

- Added tests for French, Spanish and German.

### CODE REVIEW

**Comment by aaaton** [comment]
Thank you! This is neat, but the local imports are referencing your fork `"github.com/charlesgiroux/golem/dicts"` instead of the main repo `"github.com/aaaton/golem/dicts", 

Here is a nice, short article of better forking with go: https://dev.to/loderunner/working-with-forks-in-go-3ab6

But since you're already done, maybe you could just swap back the imports to aaaton, and commit again?

File: main.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"path"
 	"strings"
 
-	"github.com/aaaton/golem/dicts"
```
Comment (aaaton) [inline main.go]: here

File: main_test.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/aaaton/golem/dicts"
```
Comment (aaaton) [inline main_test.go]: and here

**Review by aaaton — CHANGES_REQUESTED** [review]

**Comment by charlesgiroux** [comment]
Yes for sure! I'll look into doing that during the week.

**Comment by aaaton** [comment]
Thank you for your PR!

**Pull request merged**